CSharpBasics/CSharpCollections1/Address.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
```

Comment:
everything is fine, just fix the indents between the methods and you can merge it.